LICENSE

```diff
@@ -1,20 +1,21 @@
 MIT License
 
-Copyright (C) 2015-2018 CERN.
+{{ 'Copyright (C) %s %s.'|format(cookiecutter.year, cookiecutter.copyright_holder)|wordwrap(width=79)}}
```

Comment:
Actually, this is the license file of the module itself - the one that should be fixed is the ``{{cookiecutter.project_shortname}}/LICENSE``. Sorry didn't catch this in the first review.

Reply:
I fixed the other license, do you want me to revert this one (I don't see a point why not being dynamic)?